docs/rules/no-static-typos.md

```diff
@@ -0,0 +1,59 @@
+# Prevent casing typos when declaring static propTypes, contextType and defaultProps
+
+Ensure no casing typos were made declaring static class properties
+
+**Fixable:** This rule is automatically fixable using the `--fix` flag on the command line.
```

Comment:
This shouldn't be fixable - it's not safe to do so.

It's a great rule to prevent people doing this by accident - but autofixing these can cause actual behavior changes in production. eslint core's policy is to never allow autofixes that could break code.